### PR TITLE
[forge] Add metrics for insertion_to_block and block_creation_to_commit

### PR DESCRIPTION
**Pull request by hariria**
## Description

Forge provides too much granularity with 4 columns capturing metrics for all parts the blockchain latency in 4 components. The goal of this PR is to simplify this into 2 columns capturing 

1. `InsertionToBlock` -> Time from getting into the node to getting into a block
2. `BlockCreationToCommit` -> Time from getting into the block to getting into storage

### OLD
batch-pos

### New

Screenshot 2024-10-17 at 2 00 17 PM

## How Has This Been Tested?

Tested queries on grafana 👍 

- [Adhoc Forge Run before](https://github.com/aptos-labs/aptos-core/actions/runs/11264334213)
- [SUCCESS] [Adhoc Forge Run after (with `realistic_env_load_sweep`)](https://github.com/aptos-labs/aptos-core/actions/runs/11283913429)

## Evidence for latency limit changes (non-exhaustive)
- [FAILURE] [`Failed latency check, for \["P90 latency for 10 is 0.9s and exceeds limit of 0.9s"\]`](https://github.com/aptos-labs/aptos-core/actions/runs/11280108671)
- [FAILURE] [`Failed latency check, for \["P50 latency for 100 is 0.9s and exceeds limit of 0.9s"\]`](https://github.com/aptos-labs/aptos-core/actions/runs/11282998408)
- [FAILURE] [`Failed latency check, for \["P90 latency for 1000 is 1.3s and exceeds limit of 1.3s"\]`](https://github.com/aptos-labs/aptos-core/actions/runs/11282080946)

### CODE REVIEW

File: testsuite/forge-cli/src/main.rs [hunk 1]
```diff
@@ -1178,12 +1178,12 @@ fn realistic_env_load_sweep_test() -> ForgeConfig {
         test: Box::new(PerformanceBenchmark),
         workloads: Workloads::TPS(vec![10, 100, 1000, 3000, 5000, 7000]),
         criteria: [
-            (9, 0.9, 0.9, 1.2, 0),
-            (95, 0.9, 1.0, 1.2, 0),
-            (950, 1.2, 1.3, 2.0, 0),
-            (2900, 1.4, 2.2, 2.5, 0),
-            (4800, 2.0, 2.5, 3.0, 0),
-            (6700, 2.5, 3.5, 5.0, 0),
+            (9, 1.0, 1.1, 1.2, 0),
```
Comment (zekun000) [inline testsuite/forge-cli/src/main.rs:1181]: do you need to raise these for the test to pass?

File: testsuite/testcases/src/load_vs_perf_benchmark.rs [hunk 2]
```diff
@@ -482,10 +482,8 @@ fn to_table(type_name: String, results: &[Vec<SingleRunStats>]) -> Vec<String> {
         "p50 lat",
         "p90 lat",
         "p99 lat",
-        "batch->pos",
-        "pos->prop",
-        "prop->order",
-        "order->commit",
+        "insertion->block",
```
Comment (zekun000) [inline testsuite/testcases/src/load_vs_perf_benchmark.rs:485]: nit: probably better call it mempool->block

File: testsuite/forge-cli/src/main.rs (hunk 1 above)
Comment (hariria) [reply testsuite/forge-cli/src/main.rs:1181]: [`Failed latency check, for \["P90 latency for 10 is 0.9s and exceeds limit of 0.9s"\]`](https://github.com/aptos-labs/aptos-core/actions/runs/11280108671)

Yes, based on my testing via the ad-hoc forge workflow. I've also updated the PR description with the forge latency checks that failed as evidence for why I changed these numbers

**Comment by hariria** [comment]
Moved to https://github.com/aptos-labs/aptos-core/pull/15036

**Pull request closed without merging**